src/Templates/Microsoft.NetCore.VB.ProjectTemplates/ClassLibrary.vstemplate

```diff
@@ -2,7 +2,7 @@
 <VSTemplate Version="3.0.0" Type="ProjectGroup" xmlns="http://schemas.microsoft.com/developer/vstemplate/2005" xmlns:sdk="http://schemas.microsoft.com/developer/vstemplate-sdkextension/2010">
   <TemplateData>
     <Name Package="{860A27C0-B665-47F3-BC12-637E16A1050A}" ID="9"/>
-    <Description Package="{860A27C0-B665-47F3-BC12-637E16A1050A}" ID="6"/>
+    <Description Package="{860A27C0-B665-47F3-BC12-637E16A1050A}" ID="10"/>
```

Comment:
I would expect a different package here, in particular the VB package? What package is this?